youtube_dl/extractor/tiktok.py

```diff
@@ -12,6 +15,9 @@
 )
 
 
+# add to requirements.txt- bs4, newspaper, requests
```

Comment:
What is this?

Reply:
A comment regarding the requirements.txt file we talk about.
its just in order to remember adding it

Reply:
when you send PR please try not to add self written comments